lib/ReactViews/Workbench/Controls/Legend.jsx

```diff
@@ -17,6 +18,11 @@ const Legend = React.createClass({
 
     onImageError(legend) {
         legend.imageHasError = true;
+        this.forceUpdate();
```

Comment:
I'd probably use knockout computed properties for this.  If you move the `this.props.item.legendUrls.map(...)` bit into the render function, and make it look something like this:

```js
const legends = this.props.item.legendUrls.map(legendUrl => {
                 const legend = {
                     url: proxyCatalogItemUrl(this.catalogMember, legendUrl.url),
                     isImage: legendUrl.isImage(),
                     imageHasError: false,
                     onImageError: this.onImageError,
                      insertDirectly: !!legendUrl.safeSvgContent, // we only insert content we generated ourselves, not arbitrary SVG from init files.
                      safeSvgContent: {__html: legendUrl.safeSvgContent}
                  };
                  knockout.track(legend, ['imageHasError']);
                  return legend;
 })
```

The render should happen automatically if there's an error loading the image.

The only danger I can think of is the browser will try loading each legend again each time render is triggered.  The only really reliable way to fix that, though, is to put more of this state into the `LegendUrl` model object instead of keeping it in the UI.